### PR TITLE
feat: Insert Missing Contributions Migration

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
Selects all transactions of type creation that do not have a contribution and inserts a contribution for them.
Motive:
 * Get the complete the community feed for contributions
 * Simplify the query for open creations

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

**Comment by Tirokk** [comment]
I would need test data @Mogge

**Review by Elweyn — APPROVED** [review]
Checked on local and is working!
We expect 3978+270 = 4248 and getting 4250 the difference are two contributions that are deleted

**Comment by Tirokk** [comment]
I've sighted it, but haven't tested it with the data yet.
Should the testing still be done by me, or is it enough that @Aniri#5803 did it? @Mogge 

Without test I can approve.

**Comment by Mogge** [comment]
@clauspeterhuebner What do you think, can we merge this one?

File: database/migrations/0044-insert_missing_contributions.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  memo,
+  amount,
+  20 AS moderator_id,
+  502 AS confirmed_by,
+  balance_date AS confirmed_at,
+  id
+FROM
+  gradido_community.transactions
+WHERE
+  type_id = 1
```
Comment (clauspeterhuebner) [inline database/migrations/0044-insert_missing_contributions.ts:23]: der Transaction-Typ =1 bedeutet eine Gutschrift/Schöpfung, richtig?

**Review by clauspeterhuebner — APPROVED** [review]

File: database/migrations/0044-insert_missing_contributions.ts (hunk 1 above)
Comment (Mogge) [reply database/migrations/0044-insert_missing_contributions.ts:23]: Genau

**Pull request merged**